### PR TITLE
Add logArtifactLink property in RunGetLogResult.

### PR DESCRIPTION
**Pull request by huanwu**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/202/Overall-Process-of-AME-Onboarding) for the workflow.
- [ ] [Validation tools](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/84/Swagger-Validation-tools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/FAQ.md).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process]( https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review).

### CODE REVIEW

**Review by northtyphoon — APPROVED** [review]
LGTM

**Comment by huanwu** [comment]
@fengzhou-msft Could you merge the PR? All failures are "breaking changes". We just add a new property "logArtifactLink".

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json [hunk 1]
```diff
@@ -1904,6 +1904,10 @@
         "logLink": {
           "description": "The link to logs for a run on a azure container registry.",
           "type": "string"
+        },
+        "logArtifactLink": {
+          "description": "The link to logs in registry for a run on a azure container registry.",
+          "type": "string"
```
Comment (fengzhou-msft) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json:1910]: Is this a read-only property that is only used in the response?

Comment (huanwu) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json:1910]: It is only used in the response.

Comment (huanwu) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json:1910]: @fengzhou-msft Do you have any question? Could you merge?

Comment (fengzhou-msft) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json:1910]: Can you add `"readOnly": true`? It's causing a breaking change without it.

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**